{{cookiecutter.project_slug}}/k8s/local/kustomization.yaml

```
@@ -20,3 +20,5 @@ patches:
   target:
     kind: Deployment
     name: backend
+patchesStrategicMerge:
+ - ./django.yaml
```

patchesStrategicMerge is deprecated. Untested `patches` rewrite below:
```suggestion
patches:
- patch: |-
    - op: add
      path: /spec/template/spec/containers/0/env
      value:
        - name: DOCKER_GATEWAY_IP # used for debugger to reach out to host machine
          value: "172.17.0.1"
  target:
    kind: Deployment
    name: django
```